package.json

```diff
@@ -39,10 +39,11 @@
     "sequelize-encrypted": "^1.0.0"
   },
   "devDependencies": {
-    "eslint": "^4.14.0",
-    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-probot": ">=0.1.0",
-    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-import": "^2.18.2",
+    "acorn": "^6.0.0",
```

Comment:
Is `acorn` now required as a peer dep by one of our existing packages?

Reply:
It is a peer dependency of one of our dependencies, yes.

Reply:
Do you know which one? I could probably find out if I went digging through that `package-lock.json` file 😇

Reply:
Haha, sure. I only mentioned the direct parent in the PR body, which is `acorn-jsx`. The full chain is:
```
[...]/dev/work/github/integrations/slack
└─┬ eslint@4.15.0
  └─┬ espree@3.5.2
    └── acorn-jsx@3.0.1
```